src/Opdex.Platform.WebApi/Models/Requests/SnapshotFilterParameters.cs

```diff
@@ -25,7 +25,7 @@ public class SnapshotFilterParameters : FilterParameters<SnapshotCursor>
         protected override SnapshotCursor InternalBuildCursor()
         {
             if (Cursor is null) return new SnapshotCursor(Interval, StartDateTime, EndDateTime, Direction, Limit, PagingDirection.Forward, default);
-            Base64Extensions.TryBase64Decode(Cursor, out var decodedCursor);
+            Cursor.TryBase64Decode(out var decodedCursor);
```

Comment:
nitpick: Don't feel too strongly about it in this particular case, but my preference is to avoid extension methods on common types like `string`.

Reply:
Easier to read though imo, if the extension is there idk that it hurts to use it the best way we can simply because of the underlying type but 🤷‍♂️ nbd I can revert